<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GV_GraphicsViewQtWidget</class>
 <widget class="QWidget" name="GV_GraphicsViewQtWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="cornerFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>199</red>
           <green>199</green>
           <blue>199</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>199</red>
           <green>199</green>
           <blue>199</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>199</red>
           <green>199</green>
           <blue>199</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>199</red>
           <green>199</green>
           <blue>199</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="RRulerQt" name="HorizontalRuler">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation" stdset="0">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QScrollBar" name="VerticalScrollBar">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="RRulerQt" name="VerticalRuler">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation" stdset="0">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="RGraphicsViewQt" name="GraphicsView" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QScrollBar" name="HorizontalScrollBar_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="InfoLabel_2">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RRulerQt</class>
   <extends>QFrame</extends>
   <header>RRulerQt.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RGraphicsViewQt</class>
   <extends>QWidget</extends>
   <header>RGraphicsViewQt.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
